public/readme.md

```diff
@@ -0,0 +1,7 @@
+# Public
+
+This folder contains static assets served by the web server.
+
+`react-scripts` is used to serve the site during development.
```

Comment:
I think this goes to just what the structure of an application created using create-react-app looks like. Can this be in the root readme?